### DISCUSSION TITLE
[Question]IOM Connection through latest version

### DISCUSSION THREAD

**Discussion by RahulVKM96** [idea-statica/ideastatica-public#520] [Help Q&A]
Is your question related to a problem? Please describe.

Describe the solution you'd like

Describe alternatives you've considered

Additional context

**Comment by RahulVKM96** [comment]
I created an xml file from idea statica Latest Version, I wanted to read this xml via IOM functionality of Nugget Packages ,and want to open it through idea statica using external Code or using vb.net. How can i achieve this.

**Comment by nathanlukeIDEA** [comment]
Hi @RahulVKM96, Can you please explain further? Step by step what you are trying to achieve.

**Comment by RahulVKM96** [comment]
Hi @nathanlukeIDEA , 
1. I enabled Developer mode in idea statica, and clicked on model in the IOM Section so that the .xml file from the version (24.0.1.1233) can be created.
2. Now I want to Open this .xml again through this version(24.0.1.1233) via Some Code C# and VB.Net.
3. I have written this small code but is not working.
4.  -->
              Dim calcfactory As New IdeaStatica.Plugin.ConnHiddenClientFactory(IdeaInstallDir)
              Dim client As IdeaStatica.Plugin.ConnectionHiddenCheckClient = calcfactory.Create()
              client.CreateConProjFromIOM(FinalXmlFIle (#Path for xml files), FinalXmlRFIle, path)
              client.Close()
5. I have writtent the other line but can you give me an example by which i can run this .xml file via this function->
client.CreateConProjFromIOM(FinalXmlFIle (#Path for xml files), FinalXmlRFIle, path)
